jdaviz/configs/specviz2d/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -792,12 +807,15 @@ def export_bg_spectrum(self, add_data=False, **kwargs):
             Whether to add the resulting spectrum to the application, according to the options
             defined in the plugin.
         """
-        bg = self.export_bg(**kwargs)
-        spec = bg.bkg_spectrum()
+        spec = self.export_bg(**kwargs).bkg_spectrum()
 
         if add_data:
             self.bg_spec_add_results.add_results_from_plugin(spec, replace=False)
 
+        # TEMPORARY: override spectral axis to be in pixels until properly supporting plotting
```

Comment:
What issue will fix this? Is this future work?

Reply:
This is the general effort to support wavelengths (waiting for glue/bqplot work in order to sync the axes correctly).  So yes, this is intentionally added to keep the current behavior.